sidebar.js

```diff
@@ -1,22 +1,22 @@
 import './search.js'
 import { applyDefaults } from './defaults.js'
 
-// GitHub API URL for fetching the latest release
-const githubApiUrl = 'https://api.github.com/repos/hyphacoop/reader.distributed.press/releases/latest'
+// The version will now be fetched from the default.json file
+const defaultJsonUrl = './config/defaults.json'
```

Comment:
Does this work on all pages? I think we need absolute URLs or to use `import.meta.url` to generate a URL relative to the sidebar file